sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_shared/__init__.py

```diff
@@ -15,17 +15,17 @@
 from .http_challenge import HttpChallenge
 from . import http_challenge_cache as HttpChallengeCache
 
+VaultId = namedtuple("VaultId", ["vault_url", "collection", "name", "version"])
```

Comment:
Why the name change?

Reply:
`pickle` needed it to be not private, otherwise it would throw a "Can't pickle class" error